primitives/transaction/src/account/basic_account.rs

```diff
@@ -22,6 +22,14 @@ impl AccountTransactionVerification for BasicAccountVerifier {
             return Err(TransactionError::ZeroValue);
         }
 
+        if transaction.recipient_data.len() > Policy::MAX_BASIC_TX_RECIPIENT_DATA_SIZE {
+            error!(
+                "The following transaction recipient's data exceeds the maximum size:\n{:?}",
+                transaction
+            );
```

Comment:
Since you are logging the whole transaction, and not just the data, I'd reword to:
```suggestion
            error!(
                "The following transaction's recipient data exceeds the maximum size:\n{:?}",
                transaction
            );
```

Reply:
On further thought: should this even be logged in the node that processes the transaction? The transaction will be correctly rejected, so it's not an error for the node (operator), i.e. it's not unexpected behavior.

Reply:
i turned them all into warnings